src/pages/guidelines/color/implementation.mdx

```diff
@@ -552,10 +550,11 @@ Carbon color tokens (it would also be to your advantage to use the color token
 layer styles from the Carbon design asset libraries when designing).
 
 Redlining a design in one theme should be enough and not require duplicate
-designs. However, if products teams do want to have design comps in both light
-and dark mode then use this handy Sketch
-[theme switcher](https://github.com/IanKitchen-IBM/carbon-theme-switcher)
-plug-in made by an IBMer.
+designs. However, products teams may want to have design comps in multiple
+themes. In Figma, we recommend that you duplicate your screens, then swap your
```

Comment:
I believe you can only theme swap the whole file and not specific frames but we may want to check on that. 

```suggestion
themes. In Figma, we recommend that you duplicate your files, then swap your
```

Reply:
Thanks! I drafted this and wasn't 100% sure. That's probably the safer instruction.